web/source/keymanweb.js

```diff
@@ -3023,20 +3027,15 @@
                 if(typeof(util.wait) == 'function') {
                   util.wait(false);
                 }
-                var kbdName = loadingStub['KN'];
-                var lngName = loadingStub['KL'];
-                kbdName = kbdName.replace(/\s*keyboard\s*/i, '');
-
-                if(typeof(altString) == "string") {
-                  util.alert(altString, function() {
-                    keymanweb['setActiveKeyboard']('');
-                  });
-                } else {
-                  util.alert('Sorry, the '+kbdName+' keyboard for '+lngName+' is not currently available!', function() { 
-                    keymanweb['setActiveKeyboard']('');
-                  });
+
+                if(!altString) {
```

Comment:
Or could even merge it into the call below `util.alert(altString || "Sorry ..."...`